### PR TITLE
[API-51] Fixing Configs

### PR DESCRIPTION
**Pull request by QMalcolm**

### CODE REVIEW

File: config/prod.exs [hunk 1]
```diff
@@ -90,6 +89,9 @@ config :ex_aws, :s3,
   secret_access_key: System.get_env("RFG_AWS_SECRET_ACCESS_KEY"),
   region: System.get_env("RFG_API_AWS_REGION")
 
+config :ex_aws,
+  debug_requests: true
```
Comment (SuperManifolds) [inline config/prod.exs:93]: debug in prod?

Comment (SuperManifolds) [reply config/prod.exs:93]: nevermind, read the issue now. bizzare

**Review by SuperManifolds — APPROVED** [review]
lgtm

File: config/prod.exs (hunk 1 above)
Comment (QMalcolm) [reply config/prod.exs:93]: Yep. It's weird, I know. For some reason the AWS package doesn't connect to the S3 buckets properly 100% of the time unless debug is enabled...

**Pull request merged**